Dockerfiles/griddyn/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM helics as builder
+
+RUN apt update && apt install -y \
+  libboost-dev libboost-chrono-dev \
+  libboost-date-time-dev libboost-filesystem-dev \
+  libboost-program-options-dev \
+  libboost-system-dev libboost-test-dev \
+  libboost-timer-dev libsuitesparse-dev \
+  libzmq3-dev libklu1 \
+  build-essential cmake git
+
+WORKDIR /root/develop
+
+RUN git clone --single-branch -b cmake_update https://github.com/LLNL/GridDyn.git
```

Comment:
We should use a specific commit for the HELICS-Tutorial. Same for GridLAB-D, HELICS, GridDyn, psst etc.

Reply:
@kdheepak I'm assuming you mean update each Dockerfile that's cloning a git repo to use a specific branch, tag, or commit? I think I'm doing that in most, at least a specific branch... perhaps you're suggesting an actual commit hash just in case the branch goes away?

Reply:
Yes, I meant to say specific commit hash. In case the branch goes away when merged to another / is updated etc, this will still work. It gives more of a guarantee of reproducibility than just using the branch name.